src/lbaf/Execution/lbsTransferStrategyBase.py

```diff
@@ -147,7 +148,10 @@ def factory(
         try:
             # Instantiate and return object
             strategy = locals()[strategy_name + "TransferStrategy"]
-            return strategy(criterion, parameters, logger)
+            if strategy_name == "Clustering":
```

Comment:
Is there a more elegant solution than this `if` block?

Reply:
In the `develop` branch I still seem to have the following line 150:
```
 return strategy(criterion, parameters, logger)
```
at commit `0cd1502c45e683eda9963896251c7d9c87d330ac`